### DISCUSSION TITLE
Can I show bitmap picture?

### DISCUSSION THREAD

**Discussion by Yurii-huang** [Textualize/textual#4960] [Q&A] [answered]
Can I show bitmap picture? 
I know this seems a bit out of the scope of widgets, and if there are no components that support bitmap display itself, is there a better way to do it?

**Comment by TomJGooding** [comment, marked as answer]
You could try [rich-pixels](https://github.com/darrenburns/rich-pixels) to display images in Textual - see https://textual.textualize.io/FAQ/
